datacapture/src/main/java/com/google/android/fhir/datacapture/extensions/MoreQuestionnaires.kt

```diff
@@ -115,22 +99,19 @@ private fun validateLaunchContextExtension(launchExtension: Extension) {
 }
 
 /**
- * The set of supported launch contexts, as per: http://hl7.org/fhir/uv/sdc/ValueSet/launchContext
+ * Filters the provided launch contexts by matching the keys with the `code` values found in the
+ * "name" extensions.
  */
-private enum class QuestionnaireLaunchContextSet(
-  val code: String,
-  val display: String,
-  val system: String,
-  val resourceType: String,
-) {
-  PATIENT("patient", "Patient", EXTENSION_LAUNCH_CONTEXT, "Patient"),
-  ENCOUNTER("encounter", "Encounter", EXTENSION_LAUNCH_CONTEXT, "Encounter"),
-  LOCATION("location", "Location", EXTENSION_LAUNCH_CONTEXT, "Location"),
-  USER_AS_PATIENT("user", "User", EXTENSION_LAUNCH_CONTEXT, "Patient"),
-  USER_AS_PRACTITIONER("user", "User", EXTENSION_LAUNCH_CONTEXT, "Practitioner"),
-  USER_AS_PRACTITIONER_ROLE("user", "User", EXTENSION_LAUNCH_CONTEXT, "PractitionerRole"),
-  USER_AS_RELATED_PERSON("user", "User", EXTENSION_LAUNCH_CONTEXT, "RelatedPerson"),
-  STUDY("study", "ResearchStudy", EXTENSION_LAUNCH_CONTEXT, "ResearchStudy"),
+internal fun filterLaunchContextsByCodeInNameExtension(
```

Comment:
How about a shorter name like:
```suggestion
internal fun filterByCodeInNameExtension(
```

Reply:
done